vellum/workflows/triage_message/nodes/create_ticket_node.py

```diff
@@ -0,0 +1,107 @@
+import logging
+import os
+from datetime import datetime, timedelta, UTC
+from vellum.workflows.nodes import BaseNode
+from services import ActionRecord, postgres_session
+from models.job import Job
+from models.contact import Contact
+from sqlmodel import select
+from .read_message_node import ReadMessageNode
+from .triage_message_node import TriageMessageNode
+import requests
+
+logger = logging.getLogger(__name__)
+
+
+class CreateTicketNode(BaseNode):
+    contact_id = ReadMessageNode.Outputs.message["contact_id"]
+    title = TriageMessageNode.Outputs.results[0]["value"]["arguments"]["title"]
+    body = TriageMessageNode.Outputs.results[0]["value"]["arguments"]["body"]
+    repo = TriageMessageNode.Outputs.results[0]["value"]["arguments"]["repo"]
+    
+    class Outputs(BaseNode.Outputs):
+        summary: str
+    
+    def run(self) -> Outputs:
+        try:
+            github_token = os.environ.get("GITHUB_TOKEN")
```

Comment:
Instead of GitHub token env var, we need to use the GitHub private key to lease a token. Check out how our app does it and repro the logic in python